src/datagen/main/java/mekanism/common/recipe/impl/CrusherRecipeProvider.java

```diff
@@ -512,4 +517,82 @@ private void addCrusherDewaxingRecipes(RecipeOutput consumer, String basePath) {
         }
     }
+
+    private void addCrusherBioFuelRecipes(RecipeOutput consumer, String basePath) {
+        biofuel(consumer, basePath, "apple", 2, 1, Items.APPLE);
+        biofuel(consumer, basePath, "baked_potato", 2, 1, Items.BAKED_POTATO);
+        biofuel(consumer, basePath, "bamboo", 2, 1, Items.BAMBOO);
+        biofuel(consumer, basePath, "berries", 1, 1, Tags.Items.FOODS_BERRY);
+        biofuel(consumer, basePath, "big_dripleaf", 2, 1, Items.BIG_DRIPLEAF);
+        biofuel(consumer, basePath, "bread", 4, 1, Items.BREAD);
+        biofuel(consumer, basePath, "cake", 6, 1, Items.CAKE);
+        biofuel(consumer, basePath, "carved_pumpkin", 2, 1, Items.CARVED_PUMPKIN);//pumpkin minus 4 seeds (sawmill)
+        biofuel(consumer, basePath, "crimson_roots", 1, 1, Items.CRIMSON_ROOTS);
+        biofuel(consumer, basePath, "cookie", 3, 4, Items.COOKIE);
+        biofuel(consumer, basePath, "crops", 2, 1, Items.CARROT, Items.CACTUS, Items.POTATO, Items.POISONOUS_POTATO, Items.WHEAT, Items.BEETROOT, Items.NETHER_WART, Items.COCOA_BEANS);
+        biofuel(consumer, basePath, "dried_kelp", 1, 1, Items.DRIED_KELP);
+        biofuelBlock(consumer, basePath, "dried_kelp_block", 1, 1, Items.DRIED_KELP_BLOCK);
+        biofuel(consumer, basePath, "fern", 2, 1, Items.FERN);
+        biofuel(consumer, basePath, "fungus", 1, 1, Items.CRIMSON_FUNGUS, Items.WARPED_FUNGUS);
+        biofuel(consumer, basePath, "glow_lichen", 1, 1, Items.GLOW_LICHEN);
+        biofuel(consumer, basePath, "hanging_roots", 1, 1, Items.HANGING_ROOTS);
+        biofuelBlock(consumer, basePath, "hay_block", 2, 1, Items.HAY_BLOCK);
+        biofuel(consumer, basePath, "kelp", 2, 1, Items.KELP);
+        biofuel(consumer, basePath, "large_fern", 3, 1, Items.LARGE_FERN);
+        biofuel(consumer, basePath, "leaves", 1, 10, ItemTags.LEAVES);
+        biofuel(consumer, basePath, "lily_pad", 1, 1, Items.LILY_PAD);
+        biofuel(consumer, basePath, "mangrove_roots", 1, 1, Items.MANGROVE_ROOTS);
+        biofuel(consumer, basePath, "melon", 6, 1, Items.MELON);
+        biofuel(consumer, basePath, "melon_slice", 1, 1, Items.MELON_SLICE);
+        biofuel(consumer, basePath, "moss_block", 2, 1, Items.MOSS_BLOCK);
+        biofuel(consumer, basePath, "moss_carpet", 4, 3, Items.MOSS_CARPET);
+        biofuel(consumer, basePath, "mushroom_blocks", 4, 1, Items.BROWN_MUSHROOM_BLOCK, Items.RED_MUSHROOM_BLOCK, Items.MUSHROOM_STEM);
+        biofuel(consumer, basePath, "mushrooms", 1, 1, Tags.Items.MUSHROOMS);
+        biofuel(consumer, basePath, "nether_sprouts", 1, 1, Items.NETHER_SPROUTS);
+        biofuelBlock(consumer, basePath, "nether_wart_block", 2, 1, Items.NETHER_WART_BLOCK);
+        biofuel(consumer, basePath, "pitcher_pod", 1, 1, Items.PITCHER_POD);
+        final int pumpkinOutput = 6;
+        biofuel(consumer, basePath, "pumpkin", pumpkinOutput, 1, Items.PUMPKIN);
+        biofuel(consumer, basePath, "pumpkin_pie", pumpkinOutput + 1, 1, Items.PUMPKIN_PIE);
```

Comment:
Do we want to use `FOODS_PIE` instead? Or is that too broad so we just want to specify only pumpkin pie?

Reply:
based on the vanilla recipe, so no